### PR TITLE
Create 2022-03-02-incubator-lesson-spotlight-intermediate-research-so…

### PR DESCRIPTION
**Pull request by kmomar**

### CODE REVIEW

**Comment by MCMaurer** [comment]
Hi @anenadic and @steve-crouch, I've written up your submission to the Incubator Lesson Spotlight into a blog post. If you could take a quick look over it to verify that everything is correct or update anything you'd like, that would be fantastic. Let me know if you have any questions!

**Comment by anenadic** [comment]
Hi @MCMaurer - thank you very much for this! There are a couple of updates to the text:
- sentence "All the lessons have been completely written and piloted in 3 workshops." - the course has now been piloted 5 times (including one time in an industrial setting for AstraZeneca)
- sentence "We are doing final checks for language, diagram and snapshots and code output consistency, and finalising instructor notes. It is ready for reuse by the community and can be either delivered in code-along setting or as self-learning course." - code, snapshots and proofreading has been finalised, the only thing left is updating the instructor notes. I'd also change language around "can be delivered in a code-along setting" - the course works best as a self-learning course with helpers on hand to help answer any questions as people attending this course are already experienced in coding. Would you agree @steve-crouch?

File: _posts/2022/03/2022-03-02-incubator-lesson-spotlight-intermediate-research-software-development-in-python.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- Manage software improvement from feedback through agile techniques
+
+### Target Audience
+
+A typical learner is someone who is working in academic research and, needing to write some code, has gained basic software development skills either by self-learning or attending, e.g., a novice Software Carpentry Python course. They have been applying those skills in their domain of work by writing code for some time, e.g. half a year or more. However, their software development-related projects are now becoming larger and are involving more stakeholders, and they now need more intermediate software engineering skills to help them design more robust software code, automate the process of testing and verifying its correctness and support collaborations with others.
+
+## Lesson Progress
+
+All the lessons have been completely written and piloted in 3 workshops. We are doing final checks for language, diagram and snapshots and code output consistency, and finalising instructor notes. It is ready for reuse by the community and can be either delivered in code-along setting or as self-learning course.
```
Comment (MCMaurer) [inline _posts/2022/03/2022-03-02-incubator-lesson-spotlight-intermediate-research-software-development-in-python.md]: Recommend changing to:

>All the lessons have been completely written and piloted in 5 workshops, including one time in an industrial setting. We are currently working on finalising instructor notes. It is ready for reuse by the community and can be delivered in code-along setting, but we have found it works best as a self-learning course for learners with previous programming experience, with helpers in attendence to aid learners where necessary.

**Review by MCMaurer — CHANGES_REQUESTED** [review]
Just a few small changes per author request.

**Comment by MCMaurer** [comment]
Hi @anenadic, I've requested a few changes to the PR, if you want to review that language and let me know if it needs any further modification, please let me know!

File: _posts/2022/03/2022-03-02-incubator-lesson-spotlight-intermediate-research-software-development-in-python.md (hunk 1 above)
Comment (anenadic) [reply _posts/2022/03/2022-03-02-incubator-lesson-spotlight-intermediate-research-software-development-in-python.md]: Sounds good @MCMaurer - here is a small tweak + a typo fix for your consideration:

All the lessons have been completely written and piloted in 5 workshops, including one time in an industrial setting. We are currently working on finalising instructor notes. It is ready for reuse by the community and, while the course  be delivered in a code-along setting, we have found it works best as a self-learning course for learners with previous programming experience, with helpers in attendance to aid learners where necessary.

**Comment by anenadic** [comment]
> 

Small tweak suggested - otherwise good to go. Cheers @MCMaurer!

**Pull request merged**